robot-server/tests/integration/http_api/persistence/test_compatibility.py

```diff
@@ -16,21 +16,48 @@ class Snapshot:
     version: str
     expected_protocol_count: int
     expected_run_count: int
+    protocols_with_no_analyses: List[str] = field(default_factory=lambda: [])
+    runs_with_no_commands: List[str] = field(default_factory=lambda: [])
```

Comment:
Here's an alternative that avoids lambdas:

```suggestion
    protocols_with_no_analyses: List[str] = field(default_factory=list)
    runs_with_no_commands: List[str] = field(default_factory=list)
```

Purely a matter of taste. No preference from me either way.

Reply:
changed.